djhtml/tests/test_suite.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+from djhtml.modes import HTML
+
+
+class TestSuite(unittest.TestCase):
+    DIR = os.path.join(os.path.dirname(__file__), "suite")
+
+    def test_available_files(self):
+        """
+        Loop over all the files in the suite directory and compare the
+        expected output to the actual output.
+
+        """
+        for filename in os.listdir(self.DIR):
+            with self.subTest(filename):
+                if filename.endswith(".in"):
```

Comment:
I intentionally used 1 dir per case

[1] Shorter directory listing.
[2] No need to "mangle" the in-filename to get the out-filename.

Reply:
Ugh, I hate deep directory nestings. It forces me to type `cd` a lot! And the mangling has been done in a cross operating system compatible way.